templates/main.html

```diff
@@ -209,7 +209,7 @@ <h3>
     </div>
     <footer>
       <small data-i18n="gci-copyright">Google Code-in and the Google Code-in logo are trademarks of Google Inc.</small><br />
-      <small><span data-i18n="gitlab-copyright-1">The GitLab logo is licensed under</span> <a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a> <span data-i18n="gitlab-copyright-2">by GitLab Inc.</span></small>
+      <small><span data-i18n="gitlab-copyright-1">The GitLab logo is licensed under</span> <a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a> <span data-i18n="gitlab-copyright-2">by GitLab Inc.</span> <span>Built by <a href="https://coala.io/">coala</a>.</span></small>
```

Comment:
Ah, it would look better if it's in its own line.

Just copy and paste the GitLab copyright and modify it to coala.
Also add linebreak `<br>` after the GitLab copyright.